config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -837,7 +837,7 @@ periodics:
     testgrid-dashboards: sig-testing-maintenance
     testgrid-tab-name: autoupdate-patch
     description: Weekly module update to latest patched versions
-- cron: "30 17-22/5 * * 1-5"  # Bump with label `skip-review`. Run at 10:30 and 15:30 PST (17:05 UTC) Mon-Fri
+- cron: "45 17-22/5 * * 1-5"  # Bump with label `skip-review`. Run at 10:45 and 15:45 PST (17:05 UTC) Mon-Fri
```

Comment:
nit: `(17:45 and 22:45 UTC)`
Or just `(in UTC)`, the actual numbers are in the string itself

Reply:
This job determines when prow is deployed, so the timing of this job is more important than the one without label, I would prefer moving the other one back

Reply:
That makes sense. Changed it so we do the regular autobump job at 15 past the hour instead of 30